Header header

uint16[16] channels

bool ch17
bool ch18

bool frameLost
bool failsafe
